# TB Variable Site Alignment Pipeline

This pipeline has been produced to generate variable-site alignments
(consisting solely of SNVs) specifically for Mycobacterium tuberculosis, on
behalf of Caroline Colijn, Dept. of Mathematics, Imperial College London. 

It may well be applicable to other organisms but some modification may be
required to variant calling/filtering according to the genome in question.

It is designed to be run on the Imperial HPC CX1 cluster using the PBSPro
scheduling software. Modifications will be required to run in different
environments - see the installation notes in the documentation for details.
